src/index.ts

```diff
@@ -109,21 +109,45 @@ server.setRequestHandler(CallToolRequestSchema, async (request) => {
       const client = new OpenAI({
         apiKey: AI_CHAT_KEY,
         baseURL: AI_CHAT_BASE_URL,
+        timeout: 30000, // 30 second timeout for longer queries
       });
 
-      const chatCompletion = await client.chat.completions.create({
-        messages: [{ role: 'user', content: content }],
-        model: AI_CHAT_MODEL,
-      });
+      try {
+        const chatCompletion = await client.chat.completions.create({
+          messages: [
+            { role: 'system', content: 'Be precise and concise.' },
```

Comment:
Maybe the system prompt can be passed in via env vars so this isn't hardcoded into everyone's system prompt.